packages/react-day-picker/src/components/Caption/Caption.tsx

```diff
@@ -11,15 +11,18 @@ export interface CaptionProps {
   id?: string;
   /** The month where the caption is displayed. */
   displayMonth: Date;
+  /** Hide the caption label. */
+  hideLabel?: boolean;
 }
 
 /**
  * The layout of the caption:
  *
  * - `dropdown`: display dropdowns for choosing the month and the year.
  * - `buttons`: display previous month / next month buttons.
+ * - `both`: display both month / year dropdowns and previous month / next month buttons.
  */
-export type CaptionLayout = 'dropdown' | 'buttons';
+export type CaptionLayout = 'dropdown' | 'buttons' | 'both';
```

Comment:
Consider that we could have more caption layouts in the future, thus "both" would be meaningless.

I'd rename this to:

```suggestion
export type CaptionLayout = 'dropdown' | 'buttons' | 'dropdown-button';
```

Reply:
Hey, no problem, I renamed it `dropdown-buttons`